RedHatInsights/insights-content-service
# change impact number returned type

# Description
changed the returned Impact field from string to int

Fixes CCXDEV-5613

## Type of change

Please delete options that are not relevant.

- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Refactor (refactoring code, removing useless files)
- Unit tests (no changes in the code)
- Bump-up dependent library (no changes in the code)

## Testing steps

run
`./update_rules_content.sh` and than `make run` in the project root dir. Check that all static content is parsed succesfully

## Checklist
